lib/game.rb

```diff
@@ -0,0 +1,134 @@
+require_relative 'board.rb'
+require_relative 'players/human.rb'
+require_relative 'players/computer.rb'
+class Game
+  WIN_COMBINATIONS=[[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8],[0,4,8], [2,4,6]]
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1=nil, player_2=nil, board=nil)
+      board = Board.new if !board
+      player_1 = Human.new("X") if !player_1
+      player_2 = Human.new("O") if !player_2
+      
+      @board = board if board
+      @player_1 = player_1 if player_1
+      @player_2 = player_2 if player_2
+  end
+
+  def current_player
+    if (@board.turn_count % 2) == 0
+      @player_1
+    else
+      @player_2
+    end
+  end
+  
+  def over?
+
+  end
+
+  def won?
+    won=false
+    WIN_COMBINATIONS.each do |arr|
```

Comment:
You could avoid the need for the local variable `won` by changing your iterator. Here you could use [`any?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-any-3F) to return true/false based on your criteria.

```
def won?
  WIN_COMBINATIONS.any? do |arr|
    @board.cells[arr[0]] == "X" && @board.cells[arr[1]] == "X" && @board.cells[arr[2]] == "X" ||
    @board.cells[arr[0]] == "O" && @board.cells[arr[1]] == "O" && @board.cells[arr[2]] == "O"
  end
end
```